config/development.json

```diff
@@ -138,6 +138,7 @@
 		"me/account-close": true,
 		"me/account/color-scheme-picker": true,
 		"me/vat-details": true,
+		"migration_modal": false,
```

Comment:
Let's be a little more specific with the flag name, since we may be using other modals in the future, we don't know!
Maybe `migration_assistance_modal` or `migration_plan_modal`?